### PR TITLE
Update for Django 1.8 and Aldryn Reversions 1.0

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: aldryn_faq/tests/test_plugins.py [hunk 1]
```diff
@@ -86,7 +102,7 @@ def test_plugin(self):
         self.question1.save()
         request = self.get_page_request(
             page1, self.user, None, lang_code='en', edit=False)
-        context = RequestContext(request, {})
+        context = RequestContext(request, {'request': request})
```
Comment (intellectronica) [inline aldryn_faq/tests/test_plugins.py]: It's a bit strange that sometime you use a single- and sometimes a double-quote.

**Comment by intellectronica** [comment]
:+1: but why are the tests failing?

**Comment by mkoistinen** [comment]
Not sure, investigating...

**Comment by mkoistinen** [comment]
ok, here's the latest. I cannot reproduce this locally. I did find this: https://github.com/SmileyChris/easy-thumbnails/issues/368 though. We don't use Easy Thumbnails in this project, but I suspect that one of our dependencies has been hit with this issue. We don't use `select_related` anywhere in this project directly.

**Pull request merged**